tests/operators/test_agnostic_aggregate_check.py

```diff
@@ -147,16 +147,7 @@ def test_invalid_values(self):
         except ValueError:
             assert True
 
-    def test_snowflake_exact_number_of_rows(self):
-        aql.load_file(
-            path=str(self.cwd) + "/../data/homes_merge_1.csv",
-            output_table=Table(
-                "aggregate_check_test",
-                conn_id="snowflake_conn",
-                database="DWH_LEGACY",
-                schema="SANDBOX_AIRFLOW_TEST",
-            ),
-        ).operator.execute({"run_id": "foo"})
+    def test_postgres_exact_number_of_rows(self):
```

Comment:
Why did you change this from a snowflake test to a PG test?

Reply:
I saw some test which was using the PG but named snowflake, I guess I changed the wrong one. I'll revert this.

Reply:
@dimberman, Test is using Postgres table and there was a table loaded into snowflake which was never used in test. So I thought the name was misleading.